docs/program/apis/data-client.md

```diff
@@ -0,0 +1,246 @@
+- `component_type` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Type of the component used to capture the data.
+- `component_name` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Name of the component used to capture the data.
+- `method_name` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Name of the method used to capture the data.
+- `tags` [(Optional[List[str]])](https://docs.python.org/3/library/stdtypes.html#typesseq-list): Optional list of [image tags](/manage/data/label/#image-tags) to allow for tag-based data filtering when retrieving data.
+- `data_request_times` [(Optional[Tuple[datetime.datetime, datetime.datetime]])](https://docs.python.org/3/library/stdtypes.html#tuples): Optional tuple containing [`datetime`](https://docs.python.org/3/library/datetime.html) objects denoting the times this data was requested and received by the appropriate sensor.
+
+**Returns**:
+
+- [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the new file.
+
+```python {class="line-numbers linkable-line-numbers"}
+time_requested = datetime(2023, 6, 5, 11)
+time_received = datetime(2023, 6, 5, 11, 0, 3)
+
+file_id = await data_client.tabular_data_capture_upload(
+    part_id="INSERT YOUR PART ID",
+    component_type='motor',
+    component_name='left_motor',
+    method_name='IsPowered',
+    tags=["tag_1", "tag_2"],
+    data_request_times=[(time_requested, time_received)],
+    tabular_data=[{'PowerPCT': 0, 'IsPowered': False}]
+)
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient.tabular_data_capture_upload).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### FileUpload
+
+Upload arbitrary files stored on your machine to the [Viam app](https://app.viam.com) by file name.
+Uploaded files can be found under the **Files** subtab of the app's [**Data** tab](https://app.viam.com/data).
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `part_id` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Part ID of the component used to capture the data. See [Find Part ID](#find-part-id) for instructions on retrieving this value.
+- `component_type` [(Optional[str])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Type of the component used to capture the data.
+- `component_name` [(Optional[str])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Name of the component used to capture the data.
+- `method_name` [(Optional[str])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Name of the method used to capture the data.
```

Comment:
I think we want to remove this from the SDK, so can omit from the docs?

Reply:
ok!! Sounds good, thank you for pointing this out